lib/src/_shared/PickerBase.d.ts

```diff
@@ -2,7 +2,9 @@ import { MaterialUiPickersDate } from '../typings/date'
 import { DateTextFieldProps } from './ModalDialog';
 
 export interface PickerBaseProps {
-  onChange: (date: MaterialUiPickersDate) => void;
-  autoOk?: boolean;
-  ampm?: boolean;
+	onChange: (date: MaterialUiPickersDate) => void;
```

Comment:
one more little thing, can you use spaces instead tabs for indentation?

Reply:
Sure it's done